src/index.js

```diff
@@ -77,6 +78,14 @@ class Client {
     let unsupported = [];
 
     if (version) {
+      // Convert to semver (removing pathes).
+      if (!/[0-9]+\.[0-9]+\.[0-9]+/.test(version)) {
+        throw new Error(`Invalid version "${version}"`, { version });
+      }
+
+      [version] = /[0-9]+\.[0-9]+\.[0-9]+/.exec(version);
+
+      this.hasNamedParametersSupport = semver.satisfies(version, '>=0.14.0');
```

Comment:
If `version` is null, the result will be `false`, so everything will be ok here.